mail_operating_unit/models/mail_mail.py

```diff
@@ -0,0 +1,57 @@
+##############################################################################
+# Copyright (c) 2021 brain-tec AG (https://braintec.com)
+# All Rights Reserved
+#
+# Licensed under the LGPL-3.
+# See LICENSE file for full licensing details.
+##############################################################################
+
+from email.utils import formataddr
+
+from odoo import api, models
+
+
+class MailMail(models.Model):
+    _inherit = "mail.mail"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        mails = super().create(vals_list)
+
+        for mail in mails:
+            operating_unit = False
+            if mail.mail_message_id.model and mail.mail_message_id.res_id:
```

Comment:
I think here we need a previous condition: if mail.mail_message_id but test what happens if no mail message id is set and we try to access a model or res id in python3 Odoo v15

Reply:
I think is not necessary because mail_message_id is required=True